modules/monitoring-reviewing-monitoring-dashboards-admin.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * monitoring/reviewing-monitoring-dashboards.adoc
+
+[id="reviewing-monitoring-dashboards-admin_{context}"]
+= Reviewing monitoring dashboards as a cluster administrator
+
+In the *Administrator* perspective, you can view dashboards relating to core {product-title} cluster components.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. In the *Administrator* perspective in the {product-title} web console, navigate to *Monitoring* -> *Dashboards*.
+
+. Choose a dashboard in the *Dashboard* list. Some dashboards, such as *etcd* and *Prometheus* dashboards, produce additional sub-menus when selected.
+
+. Select a time range in the *Time Range* list.
```

Comment:
I think this should this say `Optional`?

Reply:
Great idea. I have now resolved this.